### PR TITLE
Add test for Batch Watch

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: ygnmi/ygnmi_test.go [hunk 1]
```diff
@@ -2361,6 +2361,134 @@ func TestBatchGet(t *testing.T) {
...
+	tests := []struct {
+		desc                 string
+		stub                 func(s *testutil.Stubber)
+		config               bool
+		paths                []ygnmi.PathStruct
+		wantSubscriptionPath []*gpb.Path
+		wantVal              *ygnmi.Value[*exampleoc.Root]
+		wantErr              string
+	}{{
+		desc: "predicate true",
```
Comment (wenovus) [inline ygnmi/ygnmi_test.go:2379]: Add an overwrite test with multiple notifications? I'm thinking can add a false->true and a true->false test, and in the latter you're checking that the false notification is ignored.

**Review by greg-dennis — APPROVED** [review]

File: ygnmi/ygnmi_test.go (hunk 1 above)
Comment (DanG100) [reply ygnmi/ygnmi_test.go:2379]: added test where pred is false, then true, then false

File: ygnmi/ygnmi_test.go [hunk 2]
```diff
@@ -2361,6 +2361,172 @@ func TestBatchGet(t *testing.T) {
...
+					Path: twoPath,
+					Val:  &gpb.TypedValue{Value: &gpb.TypedValue_StringVal{StringVal: "bar"}},
+				}},
+			}).Notification(&gpb.Notification{
+				Timestamp: 102,
+				Update: []*gpb.Update{{
+					Path: twoPath,
+					Val:  &gpb.TypedValue{Value: &gpb.TypedValue_StringVal{StringVal: "bar"}},
+				}},
+			})
```
Comment (wenovus) [inline ygnmi/ygnmi_test.go:2428]: Should the second be something other than "bar"?

Comment (DanG100) [reply ygnmi/ygnmi_test.go]: fixed thanks. the timestamp check is sufficient to verify that the last notif was ignored but this matches the description

**Review by wenovus — APPROVED** [review]

**Pull request merged**